src/api/run.rkt

```diff
@@ -111,6 +109,4 @@
   (define df
     (diff-datafiles (read-datafile (build-path old "results.json"))
                     (read-datafile (build-path new "results.json"))))
-  (call-with-output-file (build-path new "index.html")
-                         #:exists 'replace
-                         (curryr make-report-page df #f)))
+  (copy-file (web-resource "report.html") (build-path new "index.html") #t))
```

Comment:
I'm seeing this change everywhere so I'm assuming it's correct.

Reply:
Yeah, basically this file became independent of the report results after Zane rewrote the report page (report results now loaded from JSON), and I've now moved its contents into an actual static file.